src/components/layout.js

```diff
@@ -108,8 +108,8 @@ const Layout = ({
     rel: 'preconnect',
     href,
   }))
-  const scriptTags = scriptUrls.map(src => (
-    <script type="text/javascript" async="true" src={src} />
+  const scriptTags = scriptUrls.map((src, i) => (
+    <script type="text/javascript" async="true" src={src} key={i} />
```

Comment:
@ric9176 the key is used by React to identify the components in a list that is generated dynamically (example in a map functions) so it can identify those components if they are placed in a different order. That's why using the index to set the key is not good, because the index it's dependent on the order.  You always want the same key for the same component. The 'src' should be different for each script, we can use that as a key.